go.sum

```diff
@@ -169,6 +169,8 @@ github.com/armon/go-metrics v0.0.0-20190430140413-ec5e00d3c878/go.mod h1:3AMJUQh
 github.com/armon/go-metrics v0.3.0/go.mod h1:zXjbSimjXTd7vOpY8B0/2LpvNvDoXBuplAD+gJD3GYs=
 github.com/armon/go-metrics v0.3.3 h1:a9F4rlj7EWWrbj7BYw8J8+x+ZZkJeqzNyRk8hdPF+ro=
 github.com/armon/go-metrics v0.3.3/go.mod h1:4O98XIr/9W0sxpJ8UaYkvjk10Iff7SnFrb4QAOwNTFc=
```

Comment:
I'm surprised to see old versions in go.sum, but I guess linter will complain if `go mod tidy` wasn't run.

Reply:
I did run `go mod tidy`.

Reply:
Looks like bug in `go mod tidy`, there are reports about it. Let's not worry about it.